book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] There would be data duplication with this `add_rental` setup here if it is ever used. `rental` parameter represents a rental already created which would mean it already has the `book` object. But the idea is that Book should be able to create a rental with only `date`, `self` and `person` parameters. Kindly modify the `add_rental` to create a Rental object, like so:

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```
  Remember to import Rental class here for that to work.